tutorials/understanding-gov/index.md

```diff
@@ -0,0 +1,324 @@
+✔ Enter proposal vote option context: YES: XX, NO: YX, ABSTAIN: XY, NO_WITH_VETO: YY
+```
+
+Then enter the proposal deposit:
+
+```sh
+✔ Enter proposal deposit: 10stake
+Your draft proposal has successfully been generated.
+Proposals should contain off-chain metadata, please upload the metadata JSON to IPFS.
+Then, replace the generated metadata field with the IPFS CID.
+```
+
+The `draft-proposal` command has now generated two files:
+    * **draft_metadata.json**
+    * **draft_proposal.json** 
+
+The content of `draft_metadata.json` contains the information you have just entered:
+
+```json
+{
+ "title": "Test Proposal",
+ "authors": "Alice",
+ "summary": "A test proposal with simapp",
+ "details": "-",
+ "proposal_forum_url": "https://example.org/proposal/1",
+ "vote_option_context": "YES: XX, NO: YX, ABSTAIN: XY, NO_WITH_VETO: YY"
+}
+```
+
+This json is deemed to be [pinned on IPFS](https://tutorials.cosmos.network/tutorials/how-to-use-ipfs/).
+
+<HighlightBox type="note">
+
+We went ahead and pinned this file on IPFS. Its CID is `QmbmhY1eNXdmcVV8QPqV5enwLZm1mjH7iv8aYTQ4RJCH49`. You can verify its content on <https://w3s.link/ipfs/QmbmhY1eNXdmcVV8QPqV5enwLZm1mjH7iv8aYTQ4RJCH49>.
+
+</HighlightBox>
+
+Now look at the content of the generated `draft_proposal.json`:
+
+```json
+{
+ "metadata": "ipfs://CID",
+ "deposit": "10stake"
+}
+```
+
+Replace the `metadata` field by `ipfs://QmbmhY1eNXdmcVV8QPqV5enwLZm1mjH7iv8aYTQ4RJCH49`.
+
+Submit the proposal on chain from alice:
+
+```sh
+$ simd tx gov submit-proposal draft_proposal.json --from=alice --keyring-backend test
+```
+
+The command output a transaction hash. You can use it to query the proposal:
+
+```sh
+$ simd query tx 3447C74EE19EF5E4B413547F1809C1E2026A7B7A8281366F88174F37D87F7060 --output json
+```
+
+## View and vote on proposals
+
+In our case, the proposal ID is `1`. You can query the proposal with the following command:
+
+```sh
+$ simd query gov proposal 1
+```
+
+Which returns:
+
+```sh
+deposit_end_time: "2022-09-20T16:36:04.741427768Z"
+final_tally_result:
+  abstain_count: "0"
+  no_count: "0"
+  no_with_veto_count: "0"
+  yes_count: "0"
+id: "1"
+messages: []
+metadata: ipfs://QmbmhY1eNXdmcVV8QPqV5enwLZm1mjH7iv8aYTQ4RJCH49
+status: PROPOSAL_STATUS_DEPOSIT_PERIOD
+submit_time: "2022-09-18T16:36:04.741427768Z"
+total_deposit:
+- amount: "10"
+  denom: stake
+voting_end_time: null
+voting_start_time: null
+```
+
+As we can see, the proposal is in the deposit period. We cannot yet vote on it. We can see what is the minimum proposal deposit for a chain with the following command:
+
+
+```sh
+$ simd q gov params --output=json | jq .deposit_params.min_deposit
+```
+
+It returns:
+
+```json
+[
+  {
+    "denom": "stake",
+    "amount": "10000000"
+  }
+]
+```
+
+We need to deposit `9999990` stake to the proposal (remember that we have submitted the proposal with `10stake`). We can do so with the following command:
```

Comment:
```suggestion
Since you submitted the proposal with `10stake`, you need to top up the deposit with `9999990stake`. You can do so with Bob and the following command:
```